spec/views/pages/_forms.html.erb_spec.rb

```diff
@@ -56,15 +56,15 @@
     let(:is_new_page) { false }
 
     it "contains a link to add guidance" do
-      expect(rendered).to have_link(text: I18n.t("guidance.add_guidance"), href: guidance_edit_path(form_id: form.id, page_id: page.id))
+      expect(rendered).to have_link(text: I18n.t("guidance.add_guidance"), href: guidance_edit_path(form_id: 1, page_id: 2))
     end
 
     it "has no hidden field for the answer type" do
       expect(rendered).not_to have_field("question_form[answer_type]", type: :hidden)
     end
 
     it "has a delete button" do
-      expect(rendered).to have_button("delete")
+      expect(rendered).to have_link(text: I18n.t('pages.delete_question'), href: delete_page_path(form_id: form.id, page_id: page.id))
```

Comment:
This'll fix the linting:

```suggestion
      expect(rendered).to have_link(text: I18n.t("pages.delete_question"), href: delete_page_path(form_id: form.id, page_id: page.id))
```